.github/workflows/component_image.yml

```diff
@@ -35,6 +35,10 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Install required OpenSSL for rust-openssl
+        run: |
+          sudo apt-get install pkg-config libssl-dev
```

Comment:
Should't we use [rustls for the oci-client](https://github.com/oras-project/rust-oci-client/blob/c52339f0032c1b82e86fdcd9497c6e469139000e/Cargo.toml#L30)?